code/forms/DMSDocumentRequestForm.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+class DMSDocumentRequestForm extends Form
+{
+    public function __construct(Controller $controller, $name, FieldList $fields, FieldList $actions, $validator = null)
+    {
+        $requiredFields = array(
+            'ReceiverName',
+            'ReceiverPhone',
+            'ReceiverEmail',
+            'DeliveryAddressLine1',
+            'DeliveryAddressCountry',
+            'DeliveryAddressPostCode',
+        );
+        foreach ($fields as $field) {
+            if (in_array($field->name, $requiredFields)) {
+                $field->addExtraClass('requiredField');
+            }
+        }
+        $validator = new RequiredFields($requiredFields);
+
+        $actions->push(new FormAction('doRequestSend', 'Send your request'));
+
+        parent::__construct($controller, $name, $fields, $actions, $validator);
+    }
+
+    public function forTemplate()
+    {
+        return $this->renderWith(array($this->class));
+    }
+}
```

Comment:
No, the extra class isn't for the validator, it's to render the form with the DMSDocumentRequestForm.ss template. Form->setTemplate('DMSDocumentRequestForm') from the template didn't work so I went this avenue so I can override forTemplate() method.